PowerShellTools/DebugEngine/DebugServiceEventsHandlerProxy.cs

```diff
@@ -209,7 +216,14 @@ public void ClearHostScreen()
         /// <returns></returns>
         public VsKeyInfo VsReadKey()
         {
-            return Debugger.ReplWindow.WaitKey();
+            if (Debugger.ReplWindow != null)
+            {
+                return Debugger.ReplWindow.WaitKey();
+            }
+            else
+            {
+                return null;
+            }

```

Comment:
Confirm that in this case we want the PSRawHost to throw a PipelineStoppedException?